src/components/customerCases/customerCase/customerCase.module.css

```diff
@@ -8,33 +8,75 @@
 .content {
   display: flex;
   flex-direction: column;
-  max-width: 1200px;
-  gap: 2rem;
-  padding: 1rem;
 }
 
 .mainTitle {
   font-weight: 600;
+  gap: 1.62rem;
+}
+
+.mainImageWrapper {
+  width: 100%;
+  height: 36.5rem;
+  overflow: hidden;
+  padding-bottom: 2rem;
 }
 
 .mainImageWrapper img {
-  border-radius: 12px;
+  border-radius: 0.75rem;
+  width: 100%;
+  height: 100%;
+  max-height: 100%;
+  min-width: 100%;
 }
 
 .ingress {
   display: flex;
-  flex-direction: column;
-  gap: 1.5rem;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-bottom: 6rem;
+
+  @media (max-width: 1024px) {
+    flex-direction: column;
+    gap: 1.5rem;
+  }
+}
+
+.projectDescription {
+  flex: 4;
+  max-width: 40%;
+
+  @media (max-width: 1024px) {
+    max-width: 100%;
+  }
 }
 
 .projectInfo {
   display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
Project info should not be center-aligned (only has effect on mobile)

<img width="719" alt="image" src="https://github.com/user-attachments/assets/80e06d39-b190-46d0-96e4-527ac2136f32">